jsonschema-core.xml

```diff
@@ -2095,6 +2104,450 @@
+                    in section 4.2.1.  Implementations MAY deviate from this as supported by their
+                    specific languages and platforms, however it is RECOMMENDED that the output be
+                    convertible to the JSON format defined herein via serialization or other means.
+                </t>
+            </section>
+
+            <section title="Output Formats">
+                <t>
+                    This specification defines four output formats.  See the *Output Structure*
+                    section for the requirements of each format.
+                    <list>
+                        <t>
+                            Flag - A boolean which simply indicates the overall validation result
+                            with no further details.
+                        </t>
+                        <t>
+                            Basic - Provides validation information in a flat list structure.
+                        </t>
+                        <t>
+                            Detailed - Provides validation information in a condensed hierarchical
+                            structure based on the structure of the schema.
+                        </t>
+                        <t>
+                            Verbose - Provides validation information in an uncondensed hierarchical
+                            structure that matches the exact structure of the schema.
+                        </t>
+                    </list>
+                    Implementations SHOULD specify which formats of validation they support.  Each
+                    format is independent of the others.  For example, if an implementation decides
+                    to support the "detailed" format, it is not also required to support the "basic"
+                    or "flag" formats, and vice versa.
+                </t>
+
+            </section>
+
+            <section title="Minimum Information">
+                <t>
+                    Beyond the simplistic "Flag" output, additional information is useful to aid in
+                    debugging a schema or instance.  Each sub-result SHOULD contain the information
+                    contained within this section at a minimum.
+                </t>
+                <t>
+                    A single object that contains all of these components is considered an
+                    output unit.
+                </t>
+                <t>
+                    Implementations MAY elect to provide additional information.
+                </t>
+
+                <section title="Keyword Relative Location">
+                    <t>
+                        The relative location of the validating keyword that follows the validation
+                        path.  The value MUST be expressed as a JSON Pointer, and it MUST include
+                        any by-reference applicators such as "$ref" or "$recursiveRef".
```

Comment:
As in my previous now outdated comment, this will technically be a valid JSON Pointer, but it won't be resolveable when including by-reference applicators. Should we note that?

Reply:
✔️